services/app-web/src/pages/StateSubmission/ReviewSubmit/V2/ReviewSubmit/SubmissionTypeSummarySectionV2.tsx

```diff
@@ -81,59 +80,70 @@ export const SubmissionTypeSummarySectionV2 = ({
                         explainMissingData={!isSubmitted}
                         children={programNames}
                     />
-                    <DataDetail
-                        id="submissionType"
-                        label="Submission type"
-                        explainMissingData={!isSubmitted}
-                        children={
-                            SubmissionTypeRecord[
-                                contractFormData.submissionType
-                            ]
-                        }
-                    />
-                    <DataDetail
-                        id="contractType"
-                        label="Contract action type"
-                        explainMissingData={!isSubmitted}
-                        children={
-                            contractFormData.contractType
-                                ? ContractTypeRecord[
-                                      contractFormData.contractType
-                                  ]
-                                : ''
-                        }
-                    />
-                    {contractFormData.riskBasedContract !== null && (
+                    {contractFormData && (
```

Comment:
Could we exit earlier if there is no `contractFormData`. Having these loose types means we create unexpected UI states. We shouldn't be rendering any part of this pagel if theres no `contractFormData` - if there's no contract functionally and we are in a full page error.

 Take a look at `ReviewSubmitV2` for a way to handle this -` if (!contractFormData) return <GenericErrorPage />` happens early.